mycpp/gc_heap.h

```diff
@@ -207,15 +218,28 @@ class Heap {
   }
 
   void* Allocate(int num_bytes) {
+    // NOTE(Jesse): I'm not sure what we should do if we ask for 0 bytes, but
+    // this is the second bug I'm fixing due to it, so I'm putting this check
+    // and some assertions in.  Practically, we should probably crash if we
```

Comment:
Round up to some minimal allocation size perhaps? We had a minimum allocation size of 2 words. One for the header, one for... something in the object body.